registry.json

```diff
@@ -1,4 +1,19 @@
 [ 
+ {
+ "ticker": "KNGHT",
+ "poolId": "41ddaaf2303176365dc36a58483aa8aba694da490d04d84783d418e1",
+ "social": {
+  "facebook": "",
+  "twitter": "CardanoFood",
+  "website": "https://knightsofcardano.com",
+  "reddit": "",  
+  "instagram": " ",
+  "youtube": ""
+ },
+ "telegram": "KnightsOfCardano",
```

Comment:
We require a personal Telegram handle(no group or channel) this is to validate voting results.

Reply:
added personal handle

Reply:
@knghtpool I see `knghtpool` as the telegram handle. Is that your personal tg account?

Reply:
@kiriakos that's right, I did not have a personal telegram handle before setting up the pool so I chose this as my handle. I could change it to something else if preferable